lab-shiv/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "17-bearer_auth",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) Lab 17 - Bearer Auth\r ======",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Lets update these lines so we can use the DEBUG functionality and run our tests.